opm/autodiff/StandardWellsDense.hpp

```diff
@@ -829,13 +829,14 @@ namespace Opm {
 
                         if (pu.phase_used[BlackoilPhases::Vapour]) {
                             int gaspos = pu.phase_pos[BlackoilPhases::Vapour] + perf * pu.num_phases;
+                            int gaspos_well = pu.phase_pos[BlackoilPhases::Vapour] + w * pu.num_phases;
```

Comment:
there's definitely a bug, but the fix could be semantically wrong: it could be that the gas/oil rates for the perforation are wanted here. if yes, how to get these? @totto82: do we need well or perforation rates here?